server/application.js

```diff
@@ -1,55 +1,77 @@
 const express = require('express');
+// const morgan = require('morgan');
 const path = require('path');
-const db = require('../database/index');
+const db = require('../database/config');
 
 const app = express();
 
+// app.use(morgan('dev', {
+//   skip: (req, res) => res.statusCode < 400,
+//   stream: process.stderr,
+// }));
+
+// app.use(morgan('dev', {
+//   skip: (req, res) => res.statusCode >= 400,
+//   stream: process.stdout,
+// }));
+
 app.use('/restaurant/:restaurantId', express.static(path.join(__dirname, '../public/index.html')));
 app.use('/menusBundle.js', express.static(path.join(__dirname, '../public/dist/bundle.js')));
 
 app.get('/menus/restaurant/:restaurantId/menu', (req, res) => {
-  db.retrieve(req.params.restaurantId, (err, results) => {
-    if (err && err.message.includes('Cast to number failed for value "NaN"')) {
-      res.status(400).json('Bad request');
-    } else if (err) {
+  const text = `
+  SELECT r.rest_name, r.rest_headers, i.item_id, i.item_description, i.item_price, i.item_url, s.section_name, d.restriction_name
+  FROM restaurants r
+  INNER JOIN restaurant_items ri on r.rest_id = ri.rest_id
+  INNER JOIN menu_items i on ri.item_id = i.item_id
+  INNER JOIN item_section si on i.item_id = si.item_id
+  INNER JOIN menu_sections s on si.section_id = s.section_id
+  LEFT JOIN item_restrictions ir on i.item_id = ir.item_id
+  LEFT JOIN dietary_restrictions d on ir.restriction_id = d.restriction_id WHERE r.rest_id = $1;`;
```

Comment:
Just a small nitpick on the table aliases here. `restaurant_items` for `ri` makes sense but I'd change the other lines to match the same first-character-of-word scheme. Two-letter variables are useful when you're working with long strings of table names but unfortunately they can also be really confusing, especially when you have a lot of them. Consistency here is probably a good pre-emptive maintenance measure.

`restaurant_items` = `ri` looks good,
`menu_items` = `mi` instead of `i`
`item_sections` = `is` instead of the flipped `si`
`menu_sections` = `ms` instead of just `s`
`item_restrictions` = `ir` makese sense
`dietary_restrictions` is probably fine as d, but if you wanted to be fully by-the-book, I'd just suggest `dr`

Reply:
Thanks for the suggestion! Updated all these except I couldn't set item_sections to 'is' because I guess 'is' is one of those forbidden words since Postgres might use it. Ended up going with isec instead, hopefully it's clear enough.